types/navermaps/index.d.ts

```diff
@@ -1234,12 +1235,12 @@ declare namespace naver.maps {
         class HeatMap {
             constructor(heatMapOptions?: HeatMapOptions);
             getColorMap(): SpectrumStyle;
-            getData(): LatLng[] | PointArrayLiteral[];
+            getData(): LatLng[] | PointArrayLiteral[] | WeightedLocation[];
             getMap(): Map | null;
             getOptions(key?: string): HeatMapOptions;
             redraw(): void;
             setColorMap(colormap: SpectrumStyle, inReverse: boolean): void;
-            setData(data: LatLng[] | PointArrayLiteral[]): void;
+            setData(data: LatLng[] | PointArrayLiteral[] | WeightedLocation[]): void;
```

Comment:
same here
<img width="785" alt="image" src="https://user-images.githubusercontent.com/16171447/180364830-6cf3eecd-da51-4bab-8f02-2787bcb6e54f.png">

Reply:
Perhaps the document needs to be changed as well.

Reply:
loged Debugging
`console.log(heatMap.getData())`
![스크린샷 2022-07-22 오후 2 13 29](https://user-images.githubusercontent.com/46237722/180367530-8c605df3-040a-4522-9b6f-40f14c201844.png)

Reply:
Does `setData(WeightedLocation[])` work too?

Reply:
yes.
There is no problem using 'heatMap.setData()' by temporarily changing the type in my project
<img width="659" alt="스크린샷 2022-07-22 오후 2 51 53" src="https://user-images.githubusercontent.com/46237722/180372233-bac57886-355a-4dcc-a9b0-8a150b764cdb.png">
.

Reply:
![스크린샷 2022-07-22 오후 3 07 28](https://user-images.githubusercontent.com/46237722/180374263-54f34e63-1404-4a18-a311-77747e034920.png)